### PR TITLE
More examples and describing text to styles

### PR DESCRIPTION
**Pull request by Robertofon**
**What kind of change does this PR introduce?**
I wantet do answer others what I wanted to read.
Styles with PropertyEquals + Class

**What is the current behavior?**
not there

**What is the new behavior?**
My texts included

### CODE REVIEW

File: wwwroot/docs/styles/selectors.md [hunk 1]
```diff
@@ -107,6 +107,22 @@ new Style(x => x.OfType<Button>().PropertyEquals(Button.IsDefaultProperty, true)
 
 Matches any control which has the specified property set to the specified value.
 
+Not: If this specified condition is met and there is already a style with a selector with for instance `"Button"`
```
Comment (grokys) [inline wwwroot/docs/styles/selectors.md:110]: I'm not sure what the paragraph is saying. Does this explanation only apply to `PropertyEquals`, or to styles in general? If it applies to styles in general then I think this isn't the right place for it.

File: wwwroot/docs/styles/selectors.md [hunk 2]
```diff
@@ -107,6 +107,22 @@ new Style(x => x.OfType<Button>().PropertyEquals(Button.IsDefaultProperty, true)
...
+this style will be applied in addition to that style and properties will override. So, if the condition is not met anymore,
+it basically falls back to the later style. That just the hierachical add-up way the styling system works. 
+
+# PropertyEquals with Class
```
Comment (grokys) [inline wwwroot/docs/styles/selectors.md:114]: This page intends to demonstrate the individual selector components, I'm not sure this is the right place to show how to combine them as there are a _lot_ of combinations of styles...

Comment (Robertofon) [reply wwwroot/docs/styles/selectors.md:114]: Personally I was looking for how to combine class and propertyequals. Thus I thought an example might fit.
Plus how inheritance plays a role. In docs I pfefer to have redundant (simplified) explainations in the same paragraph but elsewhere (where it actually belongs) but one has to find that, first.

File: wwwroot/docs/styles/selectors.md (hunk 1 above)
Comment (Robertofon) [reply wwwroot/docs/styles/selectors.md:110]: Sorry, it should have benn a **Note** not a **Not**

**Comment by jmacato** [comment]
Inactive PR.. closing this.

**Pull request closed without merging**